### PR TITLE
[security] Add the Capabilities section

### PR DESCRIPTION
**Pull request by sgirones**
Add documentation for the new `Capabilities` concept

### CODE REVIEW

File: app/templates/docs/security/capabilities.hbs [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    <h2>Capabilities</h2>
+    <p>
+		SurrealDB is secure by default.
+		SurrealDB enables lots of use cases by offering powerful features like scripting, functions or network access from within queries.
+		At the same time, different users will have different requirements, and not all features will be necessary for all use cases.
+		Therefore SurrealDB don't allow any of those features by default and lets the administrator enable them as needed.
+	</p>
+
+	<p>When a query wants to use a feature that is not allowed, SurrealDB will reject it.</p>
+	<Code @name="docs/security/capabilities/rejected-query.surql" text="Rejected query" />
```
Comment (Ekwuno) [inline app/templates/docs/security/capabilities.hbs:17]: @sgirones this seems a bit abrupt. Can we move it to a different section or is the aim toi show what happens when accessing a feature that isn't enabled ?

File: app/templates/docs/security/capabilities.hbs [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+	<ul>
+		<li><strong>--deny-scripting</strong> Deny execution of scripting functions</li>
+		<li><strong>--deny-guests</strong> Deny non-authenticated users to execute queries when authentication is enabled</li>
+		<li><strong>--deny-funcs [&lt;target&gt;,...]</strong> Deny execution of all functions. Optionally, you can provide a comma-separated list of function names to deny.</li>
+		<li><strong>--deny-net [&lt;target&gt;,...]</strong>  Deny all outbound network access. Optionally, you can provide a comma-separated list of targets to deny.</li>
+		<li><strong>-D, --deny-all </strong> Deny all capabilities above.</li>
+	</ul>
+
+	<p>Denied capabilities will override any allowed capability that matches.</p>
+</Layout::Text>
```
Comment (Ekwuno) [inline app/templates/docs/security/capabilities.hbs:43]: @sgirones We will fix this in the docs migration but for now is it possible to have this in a table like https://surrealdb.com/docs/surrealql/functions/array

File: app/templates/docs/security/capabilities.hbs (hunk 1 above)
Comment (sgirones) [reply app/templates/docs/security/capabilities.hbs:16]: we have seen a lot of confusion regarding the behaviour of this kind of `guest queries`. People expect a 403 but receive an empty list instead. I wanted to highlight the difference depending on whether or not you allow guest access

Comment (sgirones) [reply app/templates/docs/security/capabilities.hbs:16]: In the aim of reinforcing the idea of security, I wanted to showcase how SurrealDB rejects any restricted query, but fine by me if you prefer to move it!

File: app/templates/docs/security/capabilities.hbs (hunk 2 above)
Comment (sgirones) [reply app/templates/docs/security/capabilities.hbs]: yes! @kearfy pointed it out too. changed!

File: app/templates/docs/security/capabilities.hbs (hunk 1 above)
Comment (Ekwuno) [reply app/templates/docs/security/capabilities.hbs:16]: I will try to reword it. Let me take a look again.

File: app/templates/docs/security/capabilities.hbs [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+<Layout::Text text-l text-f>
+    <h2>Capabilities</h2>
+    <p>
+		SurrealDB is secure by default and is suitable for all database use cases. It offers powerful features like scripting, functions or network access from within your SurrealQL queries.
+		
+		To aid flexibility,  SurrealDB don't enable any of those features by default and lets the administrator enable them as needed per use case. 
+	</p>
+
+	<p>When a query wants to use a feature that is not allowed, SurrealDB will reject it.</p>
+	<Code @name="docs/security/capabilities/rejected-query.surql" text="Rejected query" />
```
Comment (Ekwuno) [inline app/templates/docs/security/capabilities.hbs:16]:
```suggestion
	<p>When a query wants to use a feature that is not enabled, SurrealDB will reject it.</p>
	<Code @name="docs/security/capabilities/rejected-query.surql" text="Rejected query" />
```

**Review by Ekwuno — APPROVED** [review]
LGTM!

**Comment by Ekwuno** [comment]
@gfenn08 thanks I have fixed it.

**Pull request merged**